### ISSUE TITLE
Handles don't show for components

### ISSUE THREAD

**Issue by dvclmn** [mekkablue/ShowHandlesEverywhere#3]
Hey Rainer, 

It'd be handy if components (whether in the background, another layer, etc) could also show their handles, without requiring decomposition. Keeping components live would really help w/ workflow.

Is this possible/simple to implement?

Thanks mate,

Dave.

**Comment by mekkablue** [comment]
I tried, but it’s not as easy as I thought. Can you describe a usecase where you need the handles of components?

**Comment by dvclmn** [comment]
Understandable!

I probably use components in the background more than paths. My practice is to use Shift+Cmd+C to bring a glyph into the background that relates to the shape of the glyph I'm working on. An example being using /fi as a guide for counterspace etc when constructing other ligatures. I do like to use components, so if I update the original shape (i.e. /fi), there are much fewer steps when carrying this out to the rest, as my guide in the background has already updated etc etc.

Basically, I find myself constantly decomposing components so I can see their handles. I like to leverage the power of components in the background as a guide. Hope that helps illustrate my case. One day I'll learn more Python and this will be a pull request not a Github 'issue' :)

**Comment by mekkablue** [comment]
> using /fi as a guide for counterspace 

What I do not understand: How would the display of handles help you there?

**Comment by dvclmn** [comment]
OK, a poorly chosen example, picked because I happened to be performing that operation at the time of writing my comment. Handles didn't and wouldn't help there. But handles _not_ helping is an exception.

**Comment by mekkablue** [comment]
I added handles for components in the background. Points show for components in the foreground anyway when you approach them with the mouse.
